system-test/storage.ts

```diff
@@ -2144,19 +2144,18 @@ describe('storage', () => {
 
       const file = bucket.file('large-metadata-error-test');
       // Save file with metadata size more then 2MB.
-      await assert.rejects(
-        async () => {
-          await file.save('test', {
-            resumable: false,
+      await file.save(
```

Comment:
Let's not mix `await` with the callback usage below. You could use the `done` system, or try/catch.

Reply:
Thanks!